app/src/main/java/com/bootcamp_android/parking_app/adapter_spinner/SpinnerActivity.kt

```diff
@@ -0,0 +1,16 @@
+package com.bootcamp_android.parking_app.adapter_spinner
+
+import android.app.Activity
+import android.view.View
+import android.widget.AdapterView
+
+class SpinnerActivity:Activity(), AdapterView.OnItemSelectedListener {
```

Comment:
I'm curious about the purpose of this screen.

Reply:
I was following  the spinner tutorial in auto mode lol

Reply:
i don't  know if  it`ll be useful when the spinner works